TLM/TLM/UI/IncompatibleModsPanel.cs

```diff
@@ -231,5 +259,55 @@ public class IncompatibleModsPanel : UIPanel {
                 ValueAnimator.Animate("ModalEffect", delegate(float val) { blurEffect.opacity = val; }, new AnimatedFloat(1f, 0f, 0.7f, EasingType.CubicEaseOut), delegate() { blurEffect.Hide(); });
             }
         }
+
+        private bool DeleteLocalTMPE()
+        {
+            PluginInfo localTMPE = GetLocalTMPE();
+            try
+            {
+                Log._Debug("Attempt to Unload local TM:PE");
+                localTMPE.Unload();
+                Log._Debug("Attempt to delete local TM:PE folder " + localTMPE.modPath);
+                DirectoryUtils.DeleteDirectory(localTMPE.modPath);
+                return true;
+            }
+            catch (Exception e)
+            {
+                Log.Warning("Failed to delete local TM:PE from " + localTMPE.modPath);
+                return false;
+            }
+        }
+
+        private PluginInfo GetLocalTMPE()
+        {
+            try
+            {
+                foreach (PluginInfo plugin in Singleton<PluginManager>.instance.GetPluginsInfo())
+                {
+                    if (!plugin.isBuiltin && !plugin.isCameraScript && plugin.publishedFileID.AsUInt64 == ulong.MaxValue && GetModName(plugin).Contains("TM:PE"))
+                    {
+                        return plugin;
+                    }
+                }
+            }
+            catch (Exception e)
+            {
+                Log.Warning("IncompatibleModsWarning.GetLocalTMPE() error - see game log for details");
+                Debug.LogException(e);
+            }
+            return null;
+        }
+
+        // returns name of mod as defined in the IUserMod class of that mod
+        private string GetModName(PluginInfo pluginInfo)
```

Comment:
I need to do some testing to see if `PluginInfo.name` already contains the name - if so we won't need either.

Reply:
I think it's the name of assembly.. I mean if you change file name to e.g. TrafficManager2.dll that would be the `PluginInfo.name` (m_cachedName IIRC). `IUserMod.Name` can be useful too

Reply:
In order to resolve the issue you found with multiple local builds, I'm thinking it might be better to show local folder name rather than mod name. That way the code is cleaner, the UI better describes what is being deleted, and we can handle multiple local builds.

I just need to work out how to parse folders in a platform-agnostic manner (eg. dealing with forward vs. back slash).

Reply:
Good idea and I think you can get mod folder path from `PluginInfo`

Reply:
It now handles multiple local builds, but for some reason it's not showing correct path segment. It's like the `.Last()` is getting penultimate folder rather than last folder.

Reply:
Should now show correct path.